lib/content/audio/predictions.ex

```diff
@@ -16,12 +16,12 @@ defmodule Content.Audio.Predictions do
   @heavy_rail_routes ["Red", "Orange", "Blue"]
 
   @spec from_sign_content(
-          {Signs.Utilities.SourceConfig.source(), Content.Message.Predictions.t()},
+          Content.Message.Predictions.t(),
```

Comment:
I'm getting a Dialyzer warning here telling me that this typespec is invalid, but it's not obvious to me exactly why. Let me know if you're seeing this as well.

Reply:
I got this too (in my editor only). I think the language server's instance of Dialyzer gets confused (in a stateful way) sometimes. I was able to "fix" it by making a temporary, valid change to `Message.Predictions` that changed the type (e.g. adding field), which seemed to cause it to re-evaluate and pick up the new code.